src/seedu/addressbook/data/AddressBook.java

```diff
@@ -114,6 +114,19 @@ public UniquePersonList getAllPersons() {
         return new UniquePersonList(allPersons);
     }
 
+    /**
+     * Returns a new UniquePersonList of persons in the address book in a birthday month at the time of the call.
```

Comment:
Follow format of javadoc comments. @param @returns @throws refer https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements